[React] Lesson 1 tweaks (#437)
CodeYourFuture/syllabus: docs/fundamentals/week-3/coursework.md

- Your deployed website

Share your deployed website in your cohort's Slack channel.

## Personal development - It's time to do your research  (1h)

Task 1: 

> Set up your LinkedIn profile and let employers know that you have started your journey into the tech industry and have  built your first product! 

Reviewer: ```suggestion
Set up your LinkedIn profile. Let employers know that you have built your first product and started your journey into the tech industry.
```

Task 2: 

> It’s time to do your research, it’s time to get familiar with the industry that you want to work in. Go on google and search for roles, look at the **person specifications** that companies are hiring for, please fill out the form below with the **three soft skills** employers are currently looking for, you will be able find roles on CW Jobs, Linkedin, Indeed and any jobs board: 

Reviewer: ```suggestion
It’s time to do your research. It’s time to get familiar with the industry that you want to work in. Search for roles on Google: look at the **person specifications** that companies are hiring for.

Fill out the form below with the **three soft skills** employers are currently looking for. 

You will find roles on [CW Jobs](https://www.cwjobs.co.uk/), [Linkedin](https://www.linkedin.com/),[ Indeed](https://uk.indeed.com/),  and any jobs board.
```

(comma splices)

Reviewer: ...actually, I'd use a semi colon here :  Search for roles on Google; look at the **person specifications** that companies are hiring for.

> Employers are looking for Full stack developers that are: 

Reviewer: ```suggestion
Employers are looking for Full Stack developers that can: 
```

Here I suggest general rephrasing this from something you ARE to something you CAN DO (I know you've included the "able to" but I suggest moving this up to enforce that mindset.

1. Able to build rapport with a diverse range of people
2.
3.

> Employers are looking for Front end developers that are:

Reviewer: ```suggestion
Employers are looking for Front End developers that are:
```
either both capitalised or both small: Front End, front end

> 1.Abile to effectively present information and respond to questions from management, business and users.

Reviewer: ```suggestion
1. Effectively present information and respond to questions from management, business, and users.
```
